ossdev/__init__.py

```diff
@@ -21,51 +26,167 @@ def set(self, arr):
     def get(self):
         return self.d
 
+    def clone(self):
+        return Vector.from_arr(list(self.get()))
+
+    @property
+    def is_row(self):
+        return self.row
+
     def __len__(self):
         return len(self.d)
 
     def __repr__(self):
         return str(self.d)
 
+    def __str__(self):
+        return 'Vct%s' % self.d
+
     def __getitem__(self, item):
         return self.d[item]
 
     def __hash__(self):
         return sum(self.d)
 
     def __setitem__(self, key, value):
-        # TODO: implement
-        return None
+        if type(key) is int and type(value) is int:
+            self.d[key] = value
 
     def __cmp__(self, other):
-        # TODO: implement, -1 if self < other, 0 if self == other, 1 if self > other
-        return -1
+        this_vector = self.length()
```

Comment:
why compare by length only?

Reply:
I just chose it as a most important and natural factor. In case where the vectors are equal in length, but have different dimension amount, choosing the vector with more dimensions to be greater than the other one, even though their lengths are equal, seemed a bit wrong to me. 

Especially when you can add to a vector with less dimensions necessary dimensions and set them equal to 0. (That's what I googled)

So, that's how I went about this.
But then again, I am not super genius mathematician, it is just my opinion.
So if this is a request to implement comparing also on dimensions, I can do it.

Reply:
No need to reimplement anything :)